editor/editor.cpp

```diff
@@ -9,6 +9,7 @@
 void Editor::initialize(HWND hWnd)
 {
 	BIND_EVENT_MEMBER_FUNCTION("EditorSaveAll", Editor::saveAll);
+	BIND_EVENT_MEMBER_FUNCTION("EditorAutosave", Editor::autoSave);
```

Comment:
should be "EditorAuto**S**ave", IMO (according to the naming standards you are following.)